### PR TITLE
Add saliency_map for Semantic segmentation

### PR DESCRIPTION
**Pull request by Wovchena**
Ticket 116447

### CODE REVIEW

**Comment by Wovchena** [comment]
CC @RHeckerIntel

File: model_api/python/openvino/model_api/models/segmentation.py [hunk 1]
```diff
@@ -214,3 +219,16 @@ def postprocess(self, outputs, meta):
...
+
+def _get_activation_map(features: Union[np.ndarray, Iterable, int, float]):
+    """Getter activation_map functions."""
+    min_soft_score = np.min(features)
+    max_soft_score = np.max(features)
+    factor = 255.0 / (max_soft_score - min_soft_score + 1e-12)
+
+    float_act_map = factor * (features - min_soft_score)
+    int_act_map = np.round(float_act_map, out=float_act_map).astype(np.uint8)
+    int_act_map = cv2.applyColorMap(int_act_map, cv2.COLORMAP_JET)
```
Comment (sovrasov) [inline model_api/python/openvino/model_api/models/segmentation.py:232]: Let's remove the last two екфтыащкьы to align with other models

**Review by sovrasov — APPROVED** [review]

**Review by sovrasov — CHANGES_REQUESTED** [review]
get_activation_map is not fully consistent with Geti, as it turned out

File: model_api/python/openvino/model_api/models/segmentation.py (hunk 1 above)
Comment (Wovchena) [reply model_api/python/openvino/model_api/models/segmentation.py:232]: Do you want me to remove
```python
    int_act_map = cv2.applyColorMap(int_act_map, cv2.COLORMAP_JET)
    int_act_map = cv2.cvtColor(int_act_map, cv2.COLOR_BGR2RGB)
```
?

Comment (sovrasov) [reply model_api/python/openvino/model_api/models/segmentation.py:232]: Yes

**Review by sovrasov — APPROVED** [review]

**Pull request merged**